admin_manual/issues/general_troubleshooting.rst

```diff
@@ -294,7 +414,82 @@ Make sure to change /nextcloud to the actual subfolder your Nextcloud instance i
    files by dropping any number of leading slashes, but it does not
    do so for the use in its main configuration files.
 
-If you are running NGINX, make sure ``location = /.well-known/carddav {`` and ``location = /.well-known/caldav {`` are properly configured as described in :doc:`../installation/nginx`, adapt to use a subfolder if necessary.
+.. _service-discovery-default-well-known-configuration-label:
+
+Default handling of ``/.well-known/`` URLs by Nextcloud
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Since Nextcloud allows any app to register any service within the ``.well-known`` directory, you may want to redirect
+any request to this folder to Nextcloud.
+
+.. warning::
+   **Make sure you exclude** *well known* **services that are handled by other
+   applications,** e.g. you should *not* redirect ``/.well-known/acme-challenge`` or ``/.well-known/pki-validation/`` as they
+   are used for automated SSL certificate deployment.
+
+Also, make sure to add your ``/nextcloud/subfolder``, if your Nextcloud instance is not running in the webroot.
+
+::
+
+    <IfModule mod_rewrite.c>
```

Comment:
This configuration is done by our `.htaccess` file. 
We should not duplicate it here without a good reason.

Reply:
That's why I mentioned `# the standard redirection rules for Nextcloud`.

How about this:

```
      RewriteEngine on
      # the standard redirection rules for Nextcloud. They are already in your .htaccess file and are here just for reference
      # RewriteRule ^/\.well-known/carddav /remote.php/dav [R=301,L]
      # RewriteRule ^/\.well-known/caldav /remote.php/dav [R=301,L]
      # RewriteRule ^/\.well-known/webfinger /index.php/.well-known/webfinger [R=301,L]
      # RewriteRule ^/\.well-known/nodeinfo /index.php/.well-known/nodeinfo [R=301,L]
      ...
```

Also note, that I've left the two redirects for `webfinger` and `nodeinfo` as they are configured with a 301 permanent redirect, while the general rule below uses a 302 redirect for the reason indicated in the example's comment.

Reply:
I'm not a fan. At worst, it's outdated in a few weeks. 
Please remove the actual rewrite rules. 
Feel free to add a link to our htaccess file.